### PR TITLE
Adding Readme file for the Download Github Releases Task.

### PR DESCRIPTION
**Pull request by ashokirla**

### CODE REVIEW

File: Tasks/DownloadGitHubReleasesV0/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Download GitHub Releases
```
Comment (Lovakumar) [inline Tasks/DownloadGitHubReleasesV0/README.md]: Download GitHub Release?

File: Tasks/DownloadGitHubReleasesV0/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Overview
+
+The [Download GitHub Releases task](https://aka.ms/AA3x715) can be used to download assets from your GitHub releases as part of your CI/CD pipeline. This task works on cross platform Azure Pipeline agents running Windows, Linux or Mac and uses GitHub APIs to access GitHub releases.
```
Comment (Lovakumar) [inline Tasks/DownloadGitHubReleasesV0/README.md]: Download GitHub Release?

File: Tasks/DownloadGitHubReleasesV0/README.md (hunk 1 above)
Comment (ashokirla) [reply Tasks/DownloadGitHubReleasesV0/README.md]: The task itself is called as Download GitHub Releases. We have to be consistent.

File: Tasks/DownloadGitHubReleasesV0/README.md (hunk 2 above)
Comment (ashokirla) [reply Tasks/DownloadGitHubReleasesV0/README.md]: Same here.

File: Tasks/DownloadGitHubReleasesV0/README.md (hunk 1 above)
Comment (ashokirla) [reply Tasks/DownloadGitHubReleasesV0/README.md]: renamed as suggested

File: Tasks/DownloadGitHubReleasesV0/README.md (hunk 2 above)
Comment (ashokirla) [reply Tasks/DownloadGitHubReleasesV0/README.md]: renamed as suggested

**Pull request merged**